'FigureManager' is an undefined name in backend_wx.py

flake8 testing of https://github.com/matplotlib/matplotlib on Python 3.6.4

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./lib/matplotlib/backends/backend_wx.py:1485:18: F821 undefined name 'FigureManager'
        figmgr = FigureManager(canvas, 1, self)
                 ^
./lib/matplotlib/backends/backend_wx.py:1548:18: F821 undefined name 'FigureManager'
        figmgr = FigureManager(canvas, 1, frame)
                 ^
```

--- a/lib/matplotlib/backends/backend_webagg.py
+++ b/lib/matplotlib/backends/backend_webagg.py
@@ -45,12 +45,14 @@ class ServerThread(threading.Thread):
     def run(self):
         tornado.ioloop.IOLoop.instance().start()
 
+
 webagg_server_thread = ServerThread()
 
 
 class FigureCanvasWebAgg(core.FigureCanvasWebAggCore):
     def show(self):
         # show the figure window
+        global show  # placates pyflakes: created by @_Backend.export below
         show()
 
     def new_timer(self, *args, **kwargs):

--- a/lib/matplotlib/backends/backend_wx.py
+++ b/lib/matplotlib/backends/backend_wx.py
@@ -1456,6 +1456,7 @@ cursord = {
 @cbook.deprecated("2.2")
 class SubplotToolWX(wx.Frame):
     def __init__(self, targetfig):
+        global FigureManager  # placates pyflakes: created by @_Backend.export
         wx.Frame.__init__(self, None, -1, "Configure subplots")
 
         toolfig = Figure((6, 3))
@@ -1519,6 +1520,7 @@ class NavigationToolbar2Wx(NavigationToolbar2, wx.ToolBar):
         NavigationToolbar2.pan(self, *args)
 
     def configure_subplots(self, evt):
+        global FigureManager  # placates pyflakes: created by @_Backend.export
         frame = wx.Frame(None, -1, "Configure subplots")
 
         toolfig = Figure((6, 3))

